apps/shipments/serializers.py

```diff
@@ -318,7 +336,7 @@ class Meta:
                    'contract_version', 'device', 'updated_by', 'state', 'exception', 'delayed', 'expected_delay_hours')
 
 
-class TrackingDataSerializer(serializers.Serializer):
+class TrackingDataSerializer(rest_serializers.Serializer):
```

Comment:
Are these changes on 339 and 408 required? Not sure why they should be swapped. This one specifically should not be a rest_framework_json_api serializer because it should be parsing a `application/json` TrackingData request

Reply:
It's actually a mistake, I wanted to revert back to the previous state.